be/src/vec/functions/function_json.cpp

```diff
@@ -1131,6 +1131,319 @@ class FunctionJsonUnquote : public IFunction {
     }
 };
 
+struct FunctionJsonInsert {
+    static constexpr auto name = "json_insert";
+    static constexpr auto is_insert = true;
```

Comment:
why not use enum? boolean value is not intuitive

Reply:
I will do it.